modules/base/src/main/java/net/legacyfabric/fabric/api/util/TriState.java

```diff
@@ -15,85 +15,31 @@
  * limitations under the License.
  */
 
-package net.fabricmc.fabric.api.util;
+package net.legacyfabric.fabric.api.util;
 
-import java.util.NoSuchElementException;
 import java.util.Objects;
 import java.util.Optional;
 import java.util.function.BooleanSupplier;
 import java.util.function.Supplier;
-import java.util.stream.Stream;
 
-import javax.annotation.Nullable;
+import org.jetbrains.annotations.Nullable;
 
+/**
+ * Represents a boolean value which can be true, false or refer to a default value.
+ */
 public enum TriState {
 	/**
 	 * Represents the boolean value of {@code false}.
 	 */
-	TRUE(true) {
-		@Override
-		public TriState and(TriState other) {
-			return other == TRUE || other == DEFAULT ? TRUE : FALSE;
-		}
-
-		@Override
-		public TriState or(TriState other) {
-			return TRUE;
-		}
-
-		@Override
-		public TriState negate() {
-			return FALSE;
-		}
-	},
+	FALSE,
```

Comment:
WHy was it like that in the first place lol

Reply:
it was sponge's tristate originally